ansible_events/builtin.py

```diff
@@ -56,16 +56,20 @@ async def print_event(
     variables: Dict,
     facts: Dict,
     ruleset: str,
-    var_root: Optional[str] = None,
+    var_root: Optional[Union[str, List[str]]] = None,
```

Comment:
The type hint here can be slightly simplified as :

```Union[str, List[str], None]```

since `Optional[T]` is the same as `Union[T, None]`